plugin/Record/RecordManagerReadListViewEstate.php

```diff
@@ -248,16 +266,20 @@ private function readPicturetypesByListviewId($listviewId)
 	 *
 	 */
 
-	private function readSortbyuservaluesNyListviewId(int $listviewId)
+	public function getSortbyuservaluesByListviewId(int $listviewId) : array
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlSortbyuservalues = "SELECT `sortbyuservalue`
-				FROM {$prefix}oo_plugin_sortbyuservalues
-				WHERE `listview_id` = ".esc_sql($listviewId);
+		$sql = $pWpDb->prepare(
+			"SELECT `sortbyuservalue`
+			FROM %i
+			WHERE `listview_id` = %d",
+			$prefix."oo_plugin_sortbyuservalues",
+			esc_sql($listviewId)
```

Comment:
Hier ist kein esc_sql notwendig
```suggestion
			$listviewId
```